mail_service/worker_notify_user.py

```diff
@@ -7,24 +7,28 @@
 bs = beanstalkc.Connection(host="172.17.0.1")
 bs.watch("notify_user")
 
+LOGS_DIR = "/srv/pipeline-logs/"
+LOGS_URL_BASE = "https://registry.centos.org/pipeline-logs/"
```

Comment:
This script goes in a mail-server container, while `constant.py` is a different chroot altogether.